site/src/api/queries/appearance.ts

```diff
@@ -6,12 +6,22 @@ import { getMetadataAsJSON } from "utils/metadata";
 const initialAppearanceData = getMetadataAsJSON<AppearanceConfig>("appearance");
 const appearanceConfigKey = ["appearance"] as const;
 
-export const appearance = (): UseQueryOptions<AppearanceConfig> => {
-  return {
+export const appearance = () => {
+  const opts: UseQueryOptions<AppearanceConfig> = {
     queryKey: ["appearance"],
     initialData: initialAppearanceData,
     queryFn: () => API.getAppearance(),
   };
+  // If we have initial appearance data, we don't want to fetch
+  // the user again. We already have it!
+  if (initialAppearanceData) {
```

Comment:
I think we can remove this condition so that, when it is loaded for the first time regardless of whether it is via HTML or network, the data is cached.

Reply:
Fair actually